Source/FlowEditor/Public/Graph/FlowGraphSchema.h

```diff
@@ -52,14 +62,45 @@ class FLOWEDITOR_API UFlowGraphSchema : public UEdGraphSchema
 	virtual bool IsCacheVisualizationOutOfDate(int32 InVisualizationCacheID) const override;
 	virtual int32 GetCurrentVisualizationCacheID() const override;
 	virtual void ForceVisualizationCacheClear() const override;
+	virtual bool ArePinsCompatible(const UEdGraphPin* PinA, const UEdGraphPin* PinB, const UClass* CallingContext = nullptr, bool bIgnoreArray = false) const override;
+	virtual void ConstructBasicPinTooltip(const UEdGraphPin& Pin, const FText& PinDescription, FString& TooltipOut) const override;
+	virtual bool IsTitleBarPin(const UEdGraphPin& Pin) const override;
+	virtual bool CanShowDataTooltipForPin(const UEdGraphPin& Pin) const override;
 	// --
 
 	// FlowGraphSchema
+
+	/**
```

Comment:
These comments were from the original signatures on UEdGraphSchema_K2.  I wasn't sure if we wanted to reformat them or keep them as-is.

They aren't really "Flow Style".